src/facade/dragonfly_connection.cc

```diff
@@ -22,6 +22,8 @@
 #include "facade/memcache_parser.h"
 #include "facade/redis_parser.h"
 #include "facade/service_interface.h"
+#include "server/conn_context.h"
```

Comment:
This is circular dependency.

Reply:
you should not add server code to facade. We already discussed this.

Reply:
yes I know, I already deprecated this, I wanted to make a point :stuck_out_tongue: